projects/angular/src/accordion/_accordion.clarity.scss

```diff
@@ -118,6 +118,15 @@
     &:disabled {
       cursor: not-allowed;
 
+      .clr-accordion-status {
```

Comment:
We can put `.clr-accordion-status, .clr-accordion-title` here and de-duplicate the `color` declaration.